test/james.testing.clss.pas

```diff
@@ -44,6 +44,8 @@ interface
   {$ELSE}
     TTestCase = TestFramework.TTestCase;
   {$ENDIF}
+  ITest = James.Testing.ITest;
+  ITestSuite = James.Testing.ITestSuite;
```

Comment:
I prefer put units in a specific order here. You're putting interfaces in a unit `*.clss` which is not following the code pattern.

Could you change your PR, changing the title and code, reordering the units? :)